dc/council/code/titles/7/sections/7-1101.xml

```diff
@@ -3,15 +3,19 @@
   <num>7-1101</num>
   <heading>Authority to enter into Compact.</heading>
   <text>The Mayor of the District of Columbia is hereby authorized to enter into and execute on behalf of the District of Columbia an agreement with any state or states legally joining therein in the form substantially as set forth in this section.</text>
+  <text>
+    <center>THE INTERSTATE COMPACT ON MENTAL HEALTH</center>
+  </text>
   <para>
-    <heading>THE INTERSTATE COMPACT ON MENTAL HEALTH</heading>
-  </para>
-  <para>
-    <heading>Article I—Purpose and Findings</heading>
+    <prefix>Article</prefix>
+    <num>I</num>
+    <heading>Purpose and Findings</heading>
```

Comment:
Yes, according to the approach @dgreisen and I decided on for handling articles.